src/components/SearchResults/SearchResults.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const SearchResults = ({ artists = [] }) => {
+  const artistElements = artists.map(artist => (
```

Comment:
consider desctructuring here { artistName, artistAbout, ....} so you don't have to write "artist." so much